keyboards/hator/gravity/tkl/rules.mk

```diff
@@ -0,0 +1,2 @@
+# This file intentionally left blank
+ENCODER_MAP_ENABLE = yes
```

Comment:
```suggestion
```
This fails lint. It should be moved in keymap level.

Reply:
Furthermore, since you have `keyboard.json` in the same directory, this file is safe to remove.